ImperatorToCK3/Source/Imperator/Provinces/Pops.cpp

```diff
@@ -17,9 +18,10 @@ void Imperator::Pops::registerKeys()
 		const auto popStr = commonItems::stringOfItem(theStream).getString();
 		if (popStr.find('{') != std::string::npos)
 		{
+			Pop::Factory popFactory;
```

Comment:
Pops already has a factory as a member, no need to a new factory for each pop

Reply:
I intended to remove this one, but apparently forgot.